Tasks/Common/cloudformationutils/cloudformationutils.ts

```diff
@@ -0,0 +1,44 @@
+/*
+  Copyright 2017-2018 Amazon.com, Inc. and its affiliates. All Rights Reserved.
+  *
+  * Licensed under the MIT License. See the LICENSE accompanying this file
+  * for the specific language governing permissions and limitations under
+  * the License.
+  */
+
+import tl = require('vsts-task-lib/task');
+import CloudFormation = require('aws-sdk/clients/cloudformation');
+
+export class CloudFormationUtils {
+    // Retrieves the declared outputs of the stack and posts as either individual variables,
+    // using the output member key as variable name, or a json-formatted blob with a variable
+    // name of the stack name suffixed with 'Outputs'
+    public static async captureStackOutputs(cloudFormationClient: CloudFormation,
```

Comment:
Throughout this PR, consider replacing static class methods with top-level exported functions.

Reply:
We can do that when it's refactored please put comment on https://github.com/aws/aws-vsts-tools/pull/168